Assets/Demo/Demo.asmdef

```diff
@@ -3,7 +3,8 @@
     "references": [
         "Unity.InputSystem",
         "Unity.TextMeshPro",
-        "Steamworks.NET"
+        "Steamworks.NET",
+        "Unity.ugui"
```

Comment:
Ugh... makes me think we *really* want that separation of Unity.InputSystem and UI-related parts...

Reply:
Not for this. The Demo code references UI code directly. So now it needs to reference the UI assembly. This has _nothing_  to do with the input system.

Reply:
We might still want it, but then we'd have to do the work outlined in the PR description.

Reply:
>The Demo code references UI code directly.

Ah right, forgot about that.